src/pages/publication/SubmissionPanel.tsx

```diff
@@ -24,13 +24,17 @@ const StyledPublishButton = styled(Button)`
   margin-top: 0.5rem;
 `;
 
+enum PublishSettingFieldName {
```

Comment:
dette var vondt å si

Reply:
skal det komme mer ting her eller er det bare den ene? kanskje dumt å ha en enum for kun én verdi?

Reply:
hva med `PublicationSettings`?

Reply:
tanken var å få det likt som for andre paneler som har en enum for alle field names den komponenten har.
det er tegnet inn ett felt til på wireframes, men tror ikke det er noen planer om at det skal inn med det første